app/views/application/_search-options.html.erb

```diff
@@ -0,0 +1,23 @@
+<div id="searchOptions">
+  <ul>
+    <li><%= link_to "Everything", "#" %></li>
+    <li><%= link_to "Books", "#" %></li>
+    <li><%= link_to "Articles", "#" %></li>
+    <li><%= link_to "Databases", "#" %></li>
+  </ul>
+  <div id="advancedSearchLink">
+    <%= link_to "Advanced Search", "#" %>
+  </div>
+</div>
+
+<div id="dateTime">
+  <div id="date">
+    FRIDAY, AUGUST 17, 2018
```

Comment:
@cdoyle-temple JUst double checking that this is supposed to be hard coded for now?

Reply:
@cdoyle-temple nevermind. I see where it is pulled from